source/initial_temperature/SAVANI_perturbation.cc

```diff
@@ -176,6 +176,7 @@ namespace aspect
 
     }
 
+
```

Comment:
while you are at it, add another empty line. We usually have 1 empty line between functions in header files, and 3 empty lines between functions in .cc files.

Reply:
Thinking about this again, it is really not necessary to hold up this PR for an empty line that was missing before already. Ready to merge.